lib/image_db.rb

```diff
@@ -188,10 +208,28 @@ def full_update
         max_id = i.id
       end
 
-      logging.info "Images: #{@images.count}"
       break if data.count == 0
     end
 
+    # Twitter
+    options = { count: 200 }
+    twitter_collect_with_max_id do |t_max_id|
+      options[:max_id] = t_max_id unless t_max_id.nil?
+      print_data = { twitter: "icco", max_id: t_max_id, images: @images.count }
+      logging.info print_data.inspect
+      begin
+       ImageDB.twitter_client.favorites(options).each do |t|
+          if t.user.screen_name.eql? "archillect"
+            @images.add t.uri.to_s
+          end
+        end
```

Comment:
`end` at 225, 8 is not aligned with `ImageDB.twitter_client.favorites(options).each do |t|` at 221, 7.